src/main/java/life/qbic/portal/portlet/control/BarcodeController.java

```diff
@@ -371,9 +371,12 @@ private void reactToProjectSelection(String project) {
               bioType = s.getProperties().get("Q_SAMPLE_TYPE");
               break;
             case Q_NGS_SINGLE_SAMPLE_RUN:
-              bioType =
-                  openbis.getExperimentById2(expID).get(0).getProperties().get("Q_SEQUENCING_TYPE")
-                      + "seq";
+              String seqType = openbis.getExperimentById2(expID).get(0).getProperties().get("Q_SEQUENCING_TYPE");
+              if(seqType!=null) {
+                bioType = seqType + "seq";
+              } else {
+                bioType = "Sequencing";
+              }
```

Comment:
you could also change the check here to

```suggestion
              if(seqType==null) {
                bioType = "Sequencing";
              } else {
                bioType = seqType + "seq";
             }
```